build/features.py

```diff
@@ -942,29 +943,34 @@ def description(self):
         return "Optimization and Tuning"
 
     def enabled(self, build):
-        build.flags['optimize'] = util.get_flags(build.env, 'optimize', 1)
-        if int(build.flags['optimize']):
+        build.flags['optimize'] = SCons.ARGUMENTS.get('optimize', '')
+        if build.flags['optimize'] is not 'portable':
```

Comment:
Also, why is this enabled when optimize isn't equal to 'portable'? You want configure to run for all values of optimize so I think you should return True from this function or ```build != 'disabled'```